### PR TITLE
Onboarding process fixed

### PR DESCRIPTION
**Pull request by VarioCode**

### CODE REVIEW

File: OpenPOS-APP/Views/Onboarding/LoginScreen.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      else
+      {
+         ActivateButton(true);
+      }
+   }
+
+   private async void OnLoginButtonClicked(object sender, EventArgs e)
+   {
+      //TODO: Login auth with DB
+      //await DisplayAlert("Test", "Logging in...", "OK");
```
Comment (Tymen) [inline OpenPOS-APP/Views/Onboarding/LoginScreen.xaml.cs:47]: Unnecessary code

File: OpenPOS-APP/Views/Onboarding/LoginScreen.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+         PasswordEntry.Text = string.Empty;
+
+      }
+      else { await DisplayAlert("Oops", "You got it wrong", "Try again"); }
+
+   }
+
+   private async void CreateNewAccount_Tapped(object sender, EventArgs e)
+   {
+      // Navigation.PushAsync();
```
Comment (Tymen) [inline OpenPOS-APP/Views/Onboarding/LoginScreen.xaml.cs:66]: Remove comment if not needed

**Review by jortdr — APPROVED** [review]
All good

File: OpenPOS-APP/MainPage.xaml.cs [hunk 3]
```diff
@@ -66,14 +48,22 @@ private async void OnLoginButtonClicked(object sender, EventArgs e)
...
+			EmailEntry.Text = string.Empty;
+			PasswordEntry.Text = string.Empty;
+
 		} else { await DisplayAlert("Oops", "You got it wrong", "Try again"); }
 		
 	}
 	
 	private async void CreateNewAccount_Tapped(object sender, EventArgs e)
 	{
 		// Navigation.PushAsync();
```
Comment (Tymen) [inline OpenPOS-APP/MainPage.xaml.cs:65]: remove unused comments

**Review by Tymen — CHANGES_REQUESTED** [review]
Resolve comments

**Review by Tymen — APPROVED** [review]
Looks good!

**Pull request merged**